pyseir/inference/infer_rt.py

```diff
@@ -842,6 +865,9 @@ def infer_all(self, plot=True, shift_deaths=0):
             plt.close()
         if df_all.empty:
             logging.warning("Inference not possible for fips: %s", self.fips)
+
+        log.info("about to save df_all.csv")
+        df_all.to_csv("df_all.csv")
```

Comment:
Unclear how this doesn't overwrite every time? Debug code?

Reply:
Argh, my bad I meant to remove that